test/unit/alignment/pairwise/align_result_selector_test.cpp

```diff
@@ -63,6 +55,18 @@ TEST(alignment_selector, align_result_selector_with_list)
     }
 }
 
+TEST(alignment_selector, align_result_selector_using_score_type)
```

Comment:
... this one case II?

Reply:
I prefer single tests for single cases so I just removed the documentation from above. It just states the obvious anyway..